addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -716,7 +716,7 @@ export class OdooEditor extends EventTarget {
             if (fontSizeInput && ev.target.closest('#font-size .dropdown-toggle')) {
                 // If the click opened the font size dropdown, select the input content.
                 fontSizeInput.select();
-            } else if (!this.isSelectionInEditable()) {
+            } else if (!this.isSelectionInBlockRoot()) {
```

Comment:
Why?

Reply:
After a second check, I think what I wanted to do is checking if the selection is in the Toolbar for massmailing. But now I check the problem again, it's actually not a good fix.

The problem is that in Website, `this.document` doesn't contain the toolbar. However, in massmaling, `this.document` contains the toolbar. This means when clicking the URL in the toolbar, the `this.document.getSelection` returns different results for Website (selection is on the button) and Massmailing (selection is on the URL input). :scream_cat: 

Potential fix: add extra check if the selection is in the massmailing toolbar, if it is, don't reset the selection

Deep complete fix: make the same behavior when getting selection in Website and Massmailing :smiling_face_with_tear: